R/tm_g_patient_profile.R

```diff
@@ -955,9 +965,11 @@ srv_g_patient_profile <- function(id,
       teal.code::chunks_push_new_line()
 
       if (select_plot["lb"]) {
-        validate(
-          need(!is.null(lb_var), "Please select a lab variable.")
-        )
+        iv_lb <- shinyvalidate::InputValidator$new()
+        iv_lb$add_rule("lb_var", shinyvalidate::sv_required())
+        iv_lb$enable()
+        validate(need(iv_lb$is_valid(), "Misspecification error: please observe red flags in the interface."))
+
```

Comment:
here

Reply:
Also validation on `study days range` variable is missing in this module.

Reply:
done